lumicks/pylake/population/tests/conftest.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+import numpy as np
+import contextlib
+from lumicks.pylake.channel import Slice, Continuous
+from lumicks.pylake.population.detail.random import temp_seed
+
+
+def generate_trace(K, pi, A, mu, sigma, T, seed):
```

Comment:
Same comment about naming conventions as above. You can also see that GitHub highlights these variables as types which is not great for code consistency and readability.